clickhouse-jdbc/src/main/java/com/clickhouse/jdbc/ConnectionImpl.java

```diff
@@ -31,6 +36,46 @@ public ConnectionImpl(String url, Properties info) {
                 .build();
     }
 
+    public String getUser() {
+        return config.getUser();
+    }
+
+    public String getURL() {
+        return url;
+    }
+
+    private String getServerVersion() throws SQLException {
+        try (QueryResponse response = client.query("SELECT version()").get(30, TimeUnit.SECONDS)) {
```

Comment:
Today's implementation is calling `com.clickhouse.jdbc.internal.ClickHouseConnectionImpl#getServerInfo` to get a lot information about a server. This call is made on connect. 
Do we plan to do the same?